applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.h

```diff
@@ -199,6 +204,20 @@ int broadcast_source_start(uint8_t big_index, struct bt_le_ext_adv *ext_adv);
  */
 int broadcast_source_stop(uint8_t big_index);
 
+/**
+ * @brief	Get the broadcast ID for the given Broadcast Isochronous Group (BIG).
+ *
+ * @note	The broadcast ID is a 3 octets long value that is used to identify the broadcast.
```

Comment:
Is it three octets fixed or `at maximum`, as you write in `shell_error(shell, "Broadcast ID must be maximum 3 octets long");`, which would suggest it can also be shorter?
```suggestion
 * @note	The broadcast ID is is used to identify the broadcast. Its value is long three octets.
```